resources/install/build.xml

```diff
@@ -573,7 +573,7 @@
                 todir="${light.dir}"
                 overwrite="true" />
 
-        <property name="wix.home" value="C:\Program Files\WiX" />
+        <property name="wix.home" value="C:\wix310" />
```

Comment:
This also can be controlled with passing parameters to ant as -Dwix.home="C:\wix310".